docs/reference/modules/terraform-aws-asg/asg-instance-refresh/asg-instance-refresh.md

```diff
@@ -73,6 +76,138 @@ The Terraform [instance_refresh](https://registry.terraform.io/providers/hashico
 *   Auto Scaling Groups support up to one active instance refresh at a time. When this resource is updated, any existing refresh is cancelled.
 *   Depending on health check settings and group size, an instance refresh may take a long time or fail. The Terraform `instance_refresh` resource does not wait for the instance refresh to complete.
 
+## Sample Usage
+
+<ModuleUsage>
+
+```hcl title="main.tf"
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY GRUNTWORK'S ASG-INSTANCE-REFRESH MODULE
+# ---------------------------------------------------------------------------------------------------------------------
+
+module "asg-instance-refresh" {
+
+  source = "git::git@github.com:gruntwork-io/terraform-aws-asg.git//modules/asg-instance-refresh?ref=v0.21.1"
+
+  # ---------------------------------------------------------------------------------------------------------------------
+  # REQUIRED VARIABLES
+  # ---------------------------------------------------------------------------------------------------------------------
+
+  # The name of the ASG.
```

Comment:
NIT: should the comment specify the type? E.g., This is a `string`, whereas `desired_capacity` is a `number`.

Reply:
Captured a design story [here](https://gruntwork.atlassian.net/browse/CORE-579) to track this question. When I discussed this feature with @ebeneliason yesterday, he mentioned this as well. We decided to ship what we had and come back to this. Now we officially have a story to track the work.

Reply:
Update: I changed that story from a design story to an implementation story after discussing with Eben and Ore live.